File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/transactionTransfer.ts
import { FinPayee } from '../../shared/models/finpayee';
import { InvoiceAllocation } from '../../shared/models/invoice-allocation';
import { Transaction } from './transaction';
import { Note } from 'projects/shared-components-lib/src/lib/notes/note';

export class TransactionTransfer {
  fromDebtorAccount: FinPayee;
  transactions: Transaction[];
  toDebtorAccount: FinPayee;
  reason: string;
  requestCode: string;
  notes: Note[];
  invoiceAllocations: InvoiceAllocation[];
  fromPolicyIds: number[];
  toPolicyIds: number[];
  documentUniqueId: string;
}

File: Service/Billing/RMA.Service.Billing.Contracts/Entities/TransactionTransfer.cs
using RMA.Common.Entities;

using System.Collections.Generic;

namespace RMA.Service.Billing.Contracts.Entities
{
    public class TransactionTransfer
    {
        public FinPayee FromDebtorAccount { get; set; }
        public List<Transaction> Transactions { get; set; }
        public FinPayee ToDebtorAccount { get; set; } = new FinPayee();
        public string Reason { get; set; }
        public string RequestCode { get; set; }
        public List<Note> Notes { get; set; }
        public List<InvoiceAllocation> InvoiceAllocations { get; set; }
        public List<int> FromPolicyIds { get; set; }
        public List<int> ToPolicyIds { get; set; }
    }
}
